### DISCUSSION TITLE
How to remove .html at the end of link?

### DISCUSSION THREAD

**Discussion by georgetown-blockchain** [community/community#28922] [Other Feature Feedback, Questions, & Ideas]
Hi all,

I've looked through forums and online, but I can't seem to figure out how to change website hosted on github pages from www.example.com/about.html to www.example.com/about without completely breaking the page. FYI, I made the website on Webflow and then exported the code. 

I tried just renaming the .HTML files in the github repo to their names but no .HTML at the end, but that didn't work. 

Any help is very much appreciated!

**Comment by radar-blip** [comment]
Are you still having problems? Seems to be working now.

**Reply by georgetown-blockchain** [reply]
No, it is not working

**Reply by radar-blip** [reply]
You cant take the extension away (.html) it must remain in place.

This is your site?
https://georgetownblockchain.io/
